draft-ietf-ppm-dap.md

```diff
@@ -1214,52 +1200,48 @@ aggregation ({{aggregate-flow}}) and finalizing the aggregate shares.
 
 ## Task Configuration {#task-configuration}
 
-Prior to the start of execution of the protocol, each participant must agree on
-the configuration for each task. A task is uniquely identified by its task ID:
+A task represents a single measurement process, though potentially aggregating
+multiple, non-overlapping batches of measurements. Each participant in a task
+must agree on its configuration prior to its execution. This document does not
+specify a mechanism for distributing task parameters among participants.
+
+A task is uniquely identified by its task ID:
 
 ~~~ tls-presentation
 opaque TaskID[32];
 ~~~
 
-The task ID value MUST be a globally unique sequence of bytes. Each task has
-the following parameters associated with it:
-
-* `leader_aggregator_url`: A URL relative to which the Leader's API resources
-   can be found.
-* `helper_aggregator_url`: A URL relative to which the Helper's API resources
-  can be found.
-* The batch mode for this task (see {{batch-mode}}). This determines how reports
-  are grouped into batches and the properties that all batches for this task
-  must have. The party MUST NOT configure the task if it does not recognize the
-  batch mode.
-* `task_start`: The time from which the Clients will start uploading reports to
-  a task. Aggregators MUST reject reports with timestamps earlier than
-  `task_start` as described in {{input-share-validation}}.
-* `task_duration`: The duration of a task. The task is considered completed
-  after the end time `task_start + task_duration`. Aggregators MUST reject
-  reports that have timestamps later than the end time, and MAY choose to opt
-  out of the task if `task_duration` is too long.
```

Comment:
`task_start` and `task_duration` are collapsed into `task_interval`. I also deleted the MUST and MAYs here: these repeat prescriptions from the sections on handling uploads (leader) and aggregation jobs (helper).